sdk/python/feast/infra/passthrough_provider.py

```diff
@@ -70,7 +70,7 @@ def batch_engine(self) -> BatchMaterializationEngine:
         if self._batch_engine:
             return self._batch_engine
         else:
-            engine_config = self.repo_config._batch_engine_config
+            engine_config = self.repo_config.batch_engine_config
```

Comment:
double check this change, should be `_batch_engine_config`

Reply:
In Pydantic v2, the variables starting with underscore will be private variables and are not included in the model schema: `Attributes whose name has a leading underscore are not treated as fields by Pydantic, and are not included in the model schema. Instead, these are converted into a "private attribute" which is not validated or even set during calls to __init__, model_validate, etc.`

The original setting (in Pydantic v1), they are supposed to be included in the model schema.